docs/arcadia-users-group/20221017-conda/lesson.md

```diff
@@ -0,0 +1,369 @@
+| `conda info`     | list of information about the environment     |
+| `conda list`     | list out all the installed packages in the currently active environment    |
+| `conda remove`     | remove a conda package     |
+| `conda config --get channels`     | list out the active channels and their priorities     |
+| `conda update`     | update all the installed packages     |
+| `conda env list` | list the different environments you have set up and their locations | 
+| `conda activate mynewenv` | activate the mynewenv Conda environment (this also works for activating our `base` environment |
+
+## A note about the conda ecosystem
+
+Conda has truly been a gift to the scientific community, taking software installs that used to take days and bringing it down to minutes.
+It also simplified versioning and environment management by integrating software encoded in many different languages into its ecosystem (python, R, perl, rust...).
+It has therefore attracted contributions from many parties over the years that have lead to a diversity of modular pieces that participate in the conda ecosystem. 
+This can be especially confusing for conda newcomers.
+This section outlines many of the key pieces in the conda ecosytem to reduce confusion as people onboard to conda.
+
+Conda itself is a piece of software that is a package and environment manager.
+To get access to the conda software, you need to install it.
+The most popular way to install conda is via miniconda or miniforge.
+**[miniforge](https://github.com/conda-forge/miniforge)** is the community (conda-forge) driven minimalistic conda installer. 
+By default, subsequent package installations come from conda-forge channel.
+**[miniconda](https://docs.conda.io/en/latest/miniconda.html)** is the Anaconda (company) driven minimalistic conda installer. 
+By default, subsequent package installations come from anaconda channels (default or otherwise).
+The default channel order can be changed for either conda installation type.
+miniforge started when miniconda didn't support the linux aarch64 system architectures and was quickly adopted by many conda user.
+It's stuck around even though both miniconda and miniforge support most system architectures.
+
+One of the problems conda addresses is resolving dependency conflicts betweens many pieces of software installed in the same environment.
+This is an [NP-complete problem](https://www.anaconda.com/blog/understanding-and-improving-condas-performance) meaning it gets slower as more software is added to an environment and it's a hard problem to solve.
+[mamba](https://mamba.readthedocs.io/en/latest/index.html) is a drop-in replacement for conda that offers higher speed and more reliable environment solutions.
+However, the best way to install mamba at the moment is via conda.
+Mamba is worth the confusion it has caused -- it decreases install times by orders of magnitude thus saving time.
+
+## Extra content
```

Comment:
I think it'd be good to mention or talk about `conda-lock` https://github.com/conda-incubator/conda-lock. I think it's super neat and can dramatically speed new environment builds cross-platform